playbooks/prepare_host.yaml

```diff
@@ -270,6 +270,16 @@
         vg: vg_ceph
         lv: db
         size: 4%VG
+
+  - name: Prepare host for Ceph (disk device)
+    when:
+    - ceph_enabled
+    - ceph_devices is defined
+    block:
+    - name: Wipe filesystem from disk
+      command: wipefs -a "{{ item }}"
+      loop: "{{ ceph_devices }}"
```

Comment:
In OSP17, Ceph deployment is done via ceph-adm tool (native to ceph tooling), and we can't provide ceph_devices to it, the tool selects which disk to use for Ceph based on its properties.
Which means that in dev-install we decided to keep supporting `ceph_devices` for backward compatibility (OSP 16, etc), but in OSP17 we recommend to just set it to true; which means that this task will likely fail.

You could do:
``` yaml
- name: Prepare host for Ceph (disk device)
    when:
    - ceph_enabled
    - ceph_devices is defined
    - ceph_devices is not true
    - (ceph_devices | length) > 0
    block:
    - name: Wipe filesystem from disk
      command: wipefs -a "{{ item }}"
      loop: "{{ ceph_devices }}"
```

Reply:
Done